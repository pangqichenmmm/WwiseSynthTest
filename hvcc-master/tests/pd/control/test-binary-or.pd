#N canvas 761 150 450 300 10;
#X obj 21 19 loadbang;
#X msg 140 81 13;
#X obj 21 181 print;
#X msg 80 81 3;
#X msg 51 81 4;
#X msg 21 81 5 7;
#X obj 21 50 t b b b b b b b b b;
#X msg 280 81 5;
#X msg 235 81 200;
#X msg 200 81 -5;
#X obj 21 143 |;
#X connect 0 0 6 0;
#X connect 1 0 10 0;
#X connect 3 0 10 1;
#X connect 4 0 10 0;
#X connect 5 0 10 0;
#X connect 6 0 5 0;
#X connect 6 1 4 0;
#X connect 6 2 3 0;
#X connect 6 4 1 0;
#X connect 6 6 9 0;
#X connect 6 7 8 0;
#X connect 6 8 7 0;
#X connect 7 0 10 0;
#X connect 8 0 10 0;
#X connect 9 0 10 0;
#X connect 10 0 2 0;
