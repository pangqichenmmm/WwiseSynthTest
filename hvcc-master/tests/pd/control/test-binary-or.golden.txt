[@ 0.000] print: 5
[@ 0.000] print: 200
[@ 0.000] print: -5
[@ 0.000] print: 13
[@ 0.000] print: 7
[@ 0.000] print: 7
